algorithms/preprocessing.py

```diff
@@ -50,10 +51,19 @@ def are_twins(v0: "Vertex", v1: "Vertex"):
     :return: Boolean if vertices v0 and v1 are twins or not
     """
     if v0 in v1.neighbours:
-        v0_neighbours = v0.neighbours
+        v0_neighbours = v0.neighbours.copy()
```

Comment:
Is hier een copy nodig? Je kunt volgens mij gewoon die twee vertices uit de neighbours halen als de if statement waar is en dan returnen op 58, op regel 60 heb je niets uit die lijsten van neighbours gehaald, dus heb je volgens mij geen copy nodig.

Reply:
Je was me net voor, heb het in de nieuwe al verwijderd, kwam er inderdaad achter dat hier geen copy nodig is.